### ISSUE TITLE
Build Exception

### ISSUE THREAD

**Issue by PicoPocoo** [pererikbergman/kmp-sqldelight#1]
Hey Erik,
thanks for your cool youtube Videos!
Whenever I run ./gradlew build I do get an NP-Exception at the task: compileIosMainKotlinMetadata.
Is this reproducible on your end?
Best regards,
picopoco

**Comment by pererikbergman** [comment]
Yes, it is. I am not sure why, but it seem to be connected to SQLDelight. If I find a solution I add it.

**Comment by PicoPocoo** [comment]
There are numerous open bug tickets regarding this topic, one of the more prominent one: 
https://github.com/JetBrains/compose-multiplatform/issues/3437

**Comment by PicoPocoo** [comment]
https://youtrack.jetbrains.com/issue/KT-65761 This will propably fix the issue, but I guess that it will take some time for kmp to support kotlin versions above 2.0.X..

https://youtrack.jetbrains.com/issue/KT-65761
